### DISCUSSION TITLE
Cruiser 3.0.37

### DISCUSSION THREAD

**Discussion by devemux86** [devemux86/cruiser#406] [Announcements]
Cruiser 3.0.37 is on [Google Play](https://play.google.com/store/apps/details?id=gr.talent.cruiser):

- https://github.com/devemux86/cruiser/discussions/380

- https://github.com/devemux86/cruiser/discussions/405 (press compass)

- https://github.com/devemux86/cruiser/discussions/402 and voice navigation

<img src="https://github.com/devemux86/cruiser/assets/3484020/e6ec7fc9-7145-45b8-89a3-0850fb9b7482" width="50%"/><img src="https://github.com/devemux86/cruiser/assets/3484020/cc264cf1-6c43-4576-a356-ee5dc4a066b5" width="50%"/>

**Comment by Ciutnik** [comment]
A beautiful Christmas gift 🎄
Thank you DevSantaClaus 🎅

**Comment by devemux86** [comment]
Development of the next big features is already in progress.

We will be able to discuss them more in the near future! :slightly_smiling_face:

**Comment by 1maxi2** [comment]
Ich habe auf 3.0.37 aktualisiert. Ich habe nun gedacht wenn ich die Karte aktualisiere werden automatisch auch die Pois aktualisiert. Das scheint aber nicht der Fall zu sein siehe angefügt Screenshot.
Oder habe ich da etwas falsch verstanden?
LG 
<img src="https://github.com/devemux86/cruiser/assets/124886589/bc20de0b-053f-44da-a0dc-7f96c94f0d61" width="50%"/>

**Reply by devemux86** [reply]
@1maxi2 

Maps + POI are currently not automatically updated together.
But it could be implemented.

Karten + POI werden derzeit nicht automatisch zusammen aktualisiert.
Aber es könnte implementiert werden.

**Reply by 1maxi2** [reply]
O.k. Danke für die Info. 
Brouter muss jetzt nicht extra heruntergeladen werden da in Cruiser integriert? Hat das Auswirkungen bei schon bestehenden Kunden die ja Brouter schon früher installiert haben ?

**Reply by devemux86** [reply]
@1maxi2 

Cruiser now automatically uses the internal BRouter routing engine to make things easier.
To use the external BRouter app, you must select it in the advanced BRouter settings in Cruiser.

Cruiser verwendet jetzt automatisch die interne BRouter-Routing-Engine, um die Arbeit zu erleichtern.
Um die externe BRouter-App zu verwenden, müssen Sie sie in den erweiterten BRouter-Einstellungen in Cruiser auswählen.

**Reply by bios4** [reply]
> Maps + POI are currently not automatically updated together.
But it could be implemented.

But such a "combined" update of maps + POI was available in previous versions, wasn't it!??

> Cruiser now automatically uses the internal BRouter routing engine to make things easier.

So when using the internal version, one can delete the (previously required) external BRouter app.
Correct?

**Reply by devemux86** [reply]
> So when using the internal version, one can delete the (previously required) external BRouter app.

Correctly, if not use it with other apps.

**Comment by kds112** [comment]
> @1maxi2
> 
> Maps + POI are currently not automatically updated together. But it could be implemented.
> 
> Karten + POI werden derzeit nicht automatisch zusammen aktualisiert. Aber es könnte implementiert werden.

I propose to combine downloading, updating and activating maps and POIs in one dialogue. It would simplify the process for normal users. Advanced users could use the advanced menu settings in order to handle maps and POIs seperately.

**Reply by devemux86** [reply]
We could probably proceed in such a way.
There are 3 map providers, 2 with poi (separate vs zip), which makes things a bit complicated.

**Reply by kds112** [reply]
But not too complicated for you 😇

**Reply by MotoUKRider** [reply]
I suppose there is some expectation in the majority of *other apps* that once you've downloaded a map for your area, you should be able to search for your nearest coffee establishment without having to download a separate file and activate it within the "Search for" setting.

I've said before that although I find Cruiser excellent (and, almost) unbeatable for an adventurous, twisty multi-stop biking journey - I will use GoogleMaps when I want to get fro A to B as efficiently as possible.  Simply put, it's a better tool for the job: Easy search, traffic, and it adapts the screen when driving through a tunnel :smile: 

Annoyingly, the most flexible routing engine doesn't offer street names as you drive (unless your route has them inside), so you have to decide what kind of routing experience you want in Cruiser within the confines of the 3rd-party routing systems.

The problem with "choice", be it map type, style, search options etc, is that it often brings inconvenience: The extra download, a range of settings and/or being online before something "works" plus the inevitable question here that "it don't work" :confused: 

The fun part is finding the middle-ground for everybody!

**Reply by devemux86** [reply]
@MotoUKRider thanks, this is a nice description of the situation, we can always improve the user experience.

It is a step-by-step process, as new features are also waiting to be implemented. :slightly_smiling_face:

**Comment by Ciutnik** [comment]
I suggest changing the Polish translation of the term `Measure distance` (menu after long touching the map). Currently it is `Mierzyć dystans` but `Pomiar odległości` would be much better. By the way, this is also the name of this function in Locus 🙂

**Reply by devemux86** [reply]
@Ciutnik thanks, I updated the translation. :+1:

**Comment by devemux86** [comment]
## Happy New Year to everyone! 🎉

<img src="https://github.com/devemux86/cruiser/assets/3484020/d2d06768-373b-4f6d-88de-edc482f74046" width="50%"/>

**Reply by DolomitesTom** [reply]
happy New Year too... track details sounds good and long awaited :-)
